SUPPORT.md

```diff
@@ -0,0 +1,29 @@
+# Support
+
+## How to file issues and get help  
+
+Customers with a [support plan](https://azure.microsoft.com/en-us/support/options/) can open an Azure Support ticket [here](https://azure.microsoft.com/support/create-ticket/).
```

Comment:
`a support plan` -> `an Azure support plan`

Also, remove the word `here` and use `Azure Support ticket` as the link text. It is highly discouraged to use text without context like "here" or "this" as link text from an accessibility perspective.

Reply:
ok